tests/test_fields.py

```diff
@@ -3,6 +3,7 @@
 
 import attrs
 import pytest
+from attrs import define
```

Comment:
a nitpick: it seems none of the defines are needed

Reply:
I keep forgetting how that works :facepalm: